### ISSUE TITLE
Accessibility issues in Screen Reader experiences, missing altText and accessible name for Installed textbox, etc.,

### ISSUE THREAD

**Issue by rrelyea** [NuGet/Home#9059]
## Batch 1 - 3 fixes
 - [x] PM UI - Thumb control needs better accessibility name - don't just repeat control name.
[1049222](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1049222) - A11y_VS ASP.NET and Web Development_ Manage Nugent's-Browse_ Screen Reader : Narrator/NVDA is mentioning the vertical bar as "Thumb".

Test of fix with Accessibility Insights:
![image](https://user-images.githubusercontent.com/8865080/72912736-ef21aa80-3cf0-11ea-81fe-0ad52e5cf144.png)

- [x] PackageItemControl - Package Icon should have "Package Icon" altText
[994454](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/994454) - A11y_NuGetClient-MinorUIChanges_Installed_ScreenReader : Alternative text is not provided for "Package Icon" image for screen reader users.

Test of fix with Accessibility Insights:
![image](https://user-images.githubusercontent.com/8865080/72912940-3c9e1780-3cf1-11ea-8a2d-be63a2d0c555.png)

- [x] ProjectView - Installed textbox needs an accessibility name
[994499](https://devdiv.visualstudio.com/DevDiv/_queries/edit/994499) - A11y_NuGetClient-MinorUIChanges_TestPackage.Deprecation.CriticalBugs.AlternatePackage_Installed_AI4D: Name property does not exists for "Installed" edit box

Test of fix with Accessibility Insights:
![image](https://user-images.githubusercontent.com/8865080/72913110-7d962c00-3cf1-11ea-8402-f7b2c1d39d6e.png)

## Batch 2 - Fix deprecation section of PM UI
- [ ] Put label on icon. Make reason have label.
[994458](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/994458) - A11y_NuGetClient-MinorUIChanges_TestPackage.Deprecation.Other_Installed_ScreenReader : Narrator is announcing " edit " word unnecessarily after reading deprecated message.

Before fix:
![image](https://user-images.githubusercontent.com/8865080/72936620-6ae51c80-3d1c-11ea-988d-4452685fa154.png)

After fix:

**Comment by rrelyea** [comment]
Batch 3 isn't done... but will be moved into another issue for sprint 166.
